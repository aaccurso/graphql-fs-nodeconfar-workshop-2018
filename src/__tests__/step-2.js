const { graphql } = require('graphql');
const schema = require('../schema');
const { dirs, filesAndDirs } = require('../queries');

describe('step 2', () => {
  it('should query dirs', async () => {
    expect(await graphql({ schema, source: dirs })).toMatchSnapshot();
  });

  it('should query files and dirs', async () => {
    expect(await graphql({ schema, source: filesAndDirs })).toMatchSnapshot();
  });
});
