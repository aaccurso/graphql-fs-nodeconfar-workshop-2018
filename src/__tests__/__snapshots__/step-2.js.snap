// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`step 2 should query dirs 1`] = `
Object {
  "data": Object {
    "dirs": Array [
      Object {
        "name": "Father",
        "type": "Dir",
      },
      Object {
        "name": "Mother",
        "type": "Dir",
      },
    ],
  },
}
`;

exports[`step 2 should query files and dirs 1`] = `
Object {
  "data": Object {
    "dirs": Array [
      Object {
        "name": "Father",
        "type": "Dir",
      },
      Object {
        "name": "Mother",
        "type": "Dir",
      },
    ],
    "files": Array [
      Object {
        "name": "Bart_Simpson.png",
        "type": "File",
      },
      Object {
        "name": "Lisa_Simpson.png",
        "type": "File",
      },
      Object {
        "name": "Maggie_Simpson.png",
        "type": "File",
      },
    ],
  },
}
`;
